test/default_alias/test_default_alias.py

```diff
@@ -163,8 +163,8 @@ def test_eigh(self):
         """
         w_usp, v_usp = usp.linalg.eigh(self.arr_2d)
         w, v = scipy.linalg.eigh(self.arr_2d)
-        self.assertTrue(usp.allclose(w, w_usp))
-        self.assertTrue(usp.allclose(v, v_usp))
+        self.assertTrue(unp.allclose(w, w_usp))
+        self.assertTrue(unp.allclose(v, v_usp))
```

Comment:
Why this change?

Reply:
This code previously used `usp` instead of `unp`, but `scipy.allclose` does not exist (which would be necessary for `usp` to work). It's possible this was removed in a newer version of scipy since the last time the tests were run; otherwise I'm confused that the code here used to work.

This is also what caused your docs PR to fail.